app/src/main/java/com/github/se/icebreakrr/model/location/GeoFirestoreRepository.kt

```diff
@@ -0,0 +1,34 @@
+package com.github.se.icebreakrr.model.location
+
+import android.util.Log
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.GeoPoint
+import org.imperiumlabs.geofirestore.GeoFirestore
+import org.imperiumlabs.geofirestore.extension.setLocation
+
+class GeoFirestoreRepository(
+    private val geoFirestore: GeoFirestore =
+        GeoFirestore(FirebaseFirestore.getInstance().collection("profiles")),
+    private val auth: FirebaseAuth = FirebaseAuth.getInstance(),
```

Comment:
I have passed the auth as an argument of my MeetingRequestViewModel class because FirebaseAuth is already initialized in the main. Therefore I think it is better to initialize it in the MainActivity only and then pass it as an argument to the functions that needs it